src/client/dss.py

```diff
@@ -91,36 +128,45 @@ def parse_dss_info(printer, action, target, file):
             dss_info['bucket'] = target[0:pos]
         else:
             dss_info['bucket'] = target
-    dss_info['op'] = dss_op_from_action(action)
-    whisper("PARSED INFO:\nOP: "+ str(dss_info['op']) + "\nBucket: " + str(dss_info['bucket']) + "\nObject: " + str(dss_info['object']) + "\n")
 
-    # Reject unwanted params
-    if dss_info['bucket'] in ['*'] or dss_info['object'] in ['*']:
-        print "Wildcards not allowed in target value."
-        sys.exit(0)
 
-    if dss_info['op'] is None:
-        print "ERROR: No valid action provided for DSS service!"
-        valid_dss_actions()
-        sys.exit(0)
+def is_valid_dss_path(path):
+    # TODO: more checks to come
+    return path.startswith("dss://")
 
-    if dss_info['action'] == 'ListAllMyBuckets':
-        if len(dss_info['bucket']) > 0:
-            print "ERROR: ListAllMyBuckets only accepts \'/\' as target"
-            sys.exit(0)
+def is_valid_local_path(path):
+    # TODO: more checks to come
+    return not (path.startswith("dss://"))
+
+def validate_args():
+    global dss_info
+
+    # check for valid printer 
+    if(dss_info['printer'] not in ['curl', 'prettyprint']):
+        print "Only --curl and --prettyprint allowed as first argument!"
+        return -1
+
+    # check for valid actions
+    if(dss_info['action'] not in dss_valid_actions):
+        print dss_info['action'] + " is not a valid command"
+        return -1
+
+    # check for args based on the actions
```

Comment:
Are we validating that user is not giving object name in actions that dont take obj name? can a user download an object using **jcs dss ls dss://buck/obj** ?